.storybook/config.js

```diff
@@ -109,7 +110,8 @@ function addPlaygroundStory() {
     .addParameters({
       chromatic: {disable: true},
     })
-    .add('Playground', () => <Playground />);
+    .add('Playground', () => <Playground />)
+    .add('Kitchen Sink', () => <KitchenSink />);
```

Comment:
💅 For consistency with other component stories where the second word starts with a lowercase letter (Choice list, Button group…):
```suggestion
    .add('Kitchen sink', () => <KitchenSink />);
```